neutral_generation/constants.py

```diff
@@ -13,18 +16,18 @@
     'she': 'they',
     "she's": "they're",
     "he's": "they're",
-    'herself': 'themself',  # TODO: be careful with cases such as "Sarah herself" --> "Sarah themself"
+    'herself': 'themself',
     'himself': 'themself'
 }
 
 INJECTIVE_PRONOUNS = {
     # one-to-one mapping
-    'him': 'them',      # I talked to him --> I talked to them
-    'hers': 'theirs'    # This pen is hers --> This pen is theirs
+    'him': 'them',  # I talked to him --> I talked to them
+    'hers': 'theirs'  # This pen is hers --> This pen is theirs
 }
 
 EASY_PRONOUNS = NON_INJECTIVE_PRONOUNS.copy()
-EASY_PRONOUNS.update(INJECTIVE_PRONOUNS)  # these pronouns can be replaced with regex
+EASY_PRONOUNS.update(INJECTIVE_PRONOUNS)  # these pronouns can be replaced directly
 
 OCCUPATION_WORDS = {
```

Comment:
I'm wondering how to expand this list, or whether we want to for a baseline. If we do, two resources I've found are:

[internal] John Henry https://g3doc.corp.google.com/company/teams/societal_context/index.md?cl=head
[external] Would be nice to find something released by the non-binary gender community, but the most exhaustive list I've found is actually http://www.writinghelp-central.com/gender-neutral.html

Melvin will be able to help you navigate access permissions. It could be that John Henry is OK with your using a subset of their data in a public setting.

Reply:
I see, thanks for sharing this. I was previously using a subset of a list from Europarl, but I agree that it would be nice to find other sources for word replacements.

https://www.europarl.europa.eu/cmsdata/151780/GNL_Guidelines_EN.pdf (last 2 pages)